azure/packages/external-controller/webpack.test.js

```diff
@@ -15,14 +16,12 @@ module.exports = env => {
             extensions: [".ts", ".tsx", ".js"],
         },
         module: {
-            rules: [{
-                test: /\.tsx?$/,
-                loader: require.resolve("ts-loader")
-            },
-            {
-                test: /\.css$/i,
-                use: [require.resolve('style-loader'), require.resolve('css-loader')],
```

Comment:
Does this test not use css files?

Reply:
I don't see any CSS files in the source, and the example runs correctly locally without these plugins.